hardhat.config.ts

```diff
@@ -17,6 +20,46 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
   }
 });
 
+task("deployOffsetHelper", "Deploys and verifies OffsetHelper")
+  .addFlag("verify", "Set true to verify the OffsetHelper after deployment")
```

Comment:
Let's set this by default to `true` so we won't have to remember to set the flag when running locally and explicitly set to `false` in CI.
```suggestion
  .addFlag("verify", "Set true to verify the OffsetHelper after deployment", true)
```

Reply:
Wouldn't it make more sense to have a `dontVerify` flag instead in this case?

Reply:
Also, if we commit this the CI will fail because it'll try to verify